tcl/auth.tcl

```diff
@@ -76,7 +76,7 @@ proc qc::auth_session { session_id } {
     #| Try to authenticate user based on the session_id given
     #| Return the user_id if successful
     #| On failure throw AUTH error
-    if { [session_exists $session_id] } {
+    if { [session_valid $session_id] } {
         return [session_user_id $session_id]
```

Comment:
I think currently sessions for tlc-erp and mla-erp are non-expiring.

So the current behavior in qcode-tcl is that we only check that a session exists not whether it is valid according to timeouts and age limits.

For mla-erp sessions are expired every night by deleting all sessions from the database.

I think tlc-erp sessions never expire.

The behavior for expiring sessions was introduced of tlc-www and mla-www as custom authentication procs which is why this behavior has not made it into qcode-tcl yet.

So in short I think what you suggested is probably the desired behavior going forward.

@ bvw anything to add